### PR TITLE
Fix/attrs dims coords

### PR DESCRIPTION
**Pull request by Kobajaski**

### CODE REVIEW

File: eopf/product/store/hdf5.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ def __getitem__(self, key: str) -> "EOObject":
         from eopf.product.core import EOGroup, EOVariable
 
         obj = self._select_node(key)
+        print(dict(obj.attrs))
```
Comment (gbfree) [inline eopf/product/store/hdf5.py:75]: Bad print.

File: eopf/product/core/eo_object.py [hunk 2]
```diff
@@ -151,12 +147,11 @@ def store(self) -> Optional[EOProductStore]:
...
     def coordinates(self) -> MappingProxyType[str, Any]:
         """MappingProxyType[str, Any]: Coordinates defined by this object"""
-        dims = self.dims
         coords_group = self.product.coordinates
         retrieved_coords = {}
-        for coord_path, dim in zip(self.attrs.get(_DIMENSIONS_PATHS, ["/"] * len(dims)), dims):
-            if (coords := coords_group.get(join_eo_path(coord_path, dim))) is not None:
-                retrieved_coords[join_path(coord_path, dim)] = coords
+        for dim in self.dims:
+            if (coords := coords_group.get(dim)) is not None:
```
Comment (gbfree) [inline eopf/product/core/eo_object.py:153]: Should probably replace with 
if (coords := coords_group.get(dim)):

File: eopf/product/core/eo_variable.py [hunk 3]
```diff
@@ -61,12 +61,19 @@ def __init__(
         **kwargs: Any,
     ):
         attrs = dict(attrs) if attrs is not None else {}
+
+        from .eo_object import _DIMENSIONS_NAME
+
+        existing_dims = attrs.pop(_DIMENSIONS_NAME, [])
```
Comment (gbfree) [inline eopf/product/core/eo_variable.py:67]: Is it really a good idea to read the dims form the attr? 
Do we really want one more way to initiallize the dims of our EOvariable?

File: eopf/product/core/eo_variable.py [hunk 4]
```diff
@@ -61,12 +61,19 @@ def __init__(
...
         if not isinstance(data, (xarray.DataArray, EOVariable)) and data is not None:
             data = xarray.DataArray(data=data, name=name, attrs=attrs, **kwargs)
         if data is None:
             data = xarray.DataArray(name=name, attrs=attrs, **kwargs)
```
Comment (gbfree) [inline eopf/product/core/eo_variable.py:71]: dims are still not part of the xarray DataArray. Is it planned in a following PR, and you just hotfixed it?

File: eopf/product/store/netcdf.py [hunk 5]
```diff
@@ -65,9 +64,10 @@ def __getitem__(self, key: str) -> "EOObject":
             obj = self._select_node(key)
         except IndexError as e:  # if key is invalid, netcdf4 raise IndexError ...
             raise KeyError(e)
+        print(obj.__dict__)
```
Comment (gbfree) [inline eopf/product/store/netcdf.py:67]: Bad print.

File: eopf/product/core/eo_variable.py (hunk 3 above)
Comment (Kobajaski) [reply eopf/product/core/eo_variable.py:67]: attrs are still in attrs, so, it's currently a fix to ensure dimensions to be fully initialized when store are used.
We probably need to a proper way to work with dims.

**Pull request merged**